mitmproxy/addons/modifybody.py

```diff
@@ -0,0 +1,98 @@
+import os
+import re
+import typing
+
+from mitmproxy import exceptions
+from mitmproxy import flowfilter
+from mitmproxy import ctx
+from mitmproxy.addons.modifyheaders import parse_modify_hook
+
+
+class ModifyBody:
+    def __init__(self):
+        self.lst = []
+
+    def load(self, loader):
+        loader.add_option(
+            "modify_body", typing.Sequence[str], [],
+            """
+            Replacement pattern of the form "[/flow-filter]/regex/[@]replacement", where
+            the separator can be any character. The @ allows to provide a file path that
+            is used to read the replacement string.
+            """
+        )
+
+    def configure(self, updated):
+        """
+            .modify_body is a list of tuples (flow_filter_pattern, regex, repl):
+
+            flow_filter_pattern: a string specifying a flow filter pattern.
+            regex: a regular expression, as string.
+            repl: the replacement string
+        """
```

Comment:
Let me be a bit pendantic here - this comment doesn't make too much sense to me:

 - `.modify_body` would indicate that there's a modify_body attribute. As far as I can see, there isn't?
 - If we want to document the type of `.lst` (that thing could probably be renamed, yes...), it would make sense to do this at the definition and not here. I'll add some ideas on how we can do this better in the next comment.

Reply:
I agree. This comment existed in the existing `Replace` addon and I didn't change it properly.